addons/web/static/src/js/chrome/abstract_web_client.js

```diff
@@ -120,16 +120,16 @@ var AbstractWebClient = Widget.extend(KeyboardNavigationMixin, {
         var state = $.bbq.getState();
         // If not set on the url, retrieve cids from the local storage
         // of from the default company on the user
-        var current_company_id = session.user_companies.current_company[0]
+        var userCompanyId = session.user_company[0];
+        var userCompanyIDS = _.map(session.user_companies, c => c[0]);
         if (!state.cids) {
-            state.cids = utils.get_cookie('cids') !== null ? utils.get_cookie('cids') : String(current_company_id);
+            state.cids = utils.get_cookie('cids') !== null ? utils.get_cookie('cids') : String(userCompanyId);
         }
-        var stateCompanyIDS = _.map(state.cids.split(','), function (cid) { return parseInt(cid) });
-        var userCompanyIDS = _.map(session.user_companies.allowed_companies, function(company) {return company[0]});
+        var stateCompanyIDS = _.map(state.cids.split(','), cid => parseInt(cid));
```

Comment:
You don't have to change it for this commit (because we're currently re-writing the webclient anyway), but for further JS developments in master:
 - always use `const` (instead of var), and `let` when you can't
 - try not to use _ anymore (here you could have used the native `map` function on arrays)